CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## UNRELEASED
 
 BUG FIXES:
+* Control Plane
+  * Add go-discover binary to control-plane image [[GH-1749](https://github.com/hashicorp/consul-k8s/pull/1749)]
```

Comment:
nit: should this say:
```suggestion
  * Add discover binary to control-plane image [[GH-1749](https://github.com/hashicorp/consul-k8s/pull/1749)]
```

since iirc that is the name of the binary?